### PR TITLE
fixes #2667 - cleanup deprecation warnings

### LINKED ISSUE #2667: Many deprecation warnings in 5.2
If running tests there are many deprecation warnings in Products.CMFPlone.

### PR DESCRIPTION
**Pull request by jensens**

### CODE REVIEW

File: Products/CMFPlone/tests/testSyndication.py [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
+class BaseSyndicationTest(PloneTestCase.PloneTestCase):
     def afterSetUp(self):
-        self.syndication = getToolByName(self.portal, 'portal_syndication')
-        self.folder.invokeFactory('Document', 'doc1')
-        self.folder.invokeFactory('Document', 'doc2')
-        self.folder.invokeFactory('File', 'file')
+        self.syndication = getToolByName(self.portal, "portal_syndication")
+        self.folder.invokeFactory("Document", "doc1")
+        self.folder.invokeFactory("Document", "doc2")
+        self.folder.invokeFactory("File", "file")
```
Comment (mauritsvanrees) [inline Products/CMFPlone/tests/testSyndication.py]: Huh? Don't we prefer single quotes?

File: pyproject.toml [hunk 2]
```diff
@@ -18,3 +18,7 @@ showcontent = true
 directory = "bugfix"
 name = "Bug fixes:"
 showcontent = true
+
+[tool.black]
```
Comment (mauritsvanrees) [inline pyproject.toml]: I haven't seen this one yet.
Note to self: read https://black.readthedocs.io/en/stable/index.html
Is this what caused the change from single to double quotes though?

Comment (mauritsvanrees) [reply pyproject.toml]: Apparently: yes. See rationale at https://black.readthedocs.io/en/stable/the_black_code_style.html#strings.
cc @gforcada

Comment (ale-rt) [reply pyproject.toml]: That's one of the reason I prefer https://github.com/google/yapf over black.

Comment (gforcada) [reply pyproject.toml]: I, and we are using that at work, would run black with the -S switch and keep the flake8 pluguins we are using for keeping the single quotes consistency

File: Products/CMFPlone/tests/testSyndication.py (hunk 1 above)
Comment (jensens) [reply Products/CMFPlone/tests/testSyndication.py]: huh, seems I used black with wrong params here. yeah, I would love to go to black defaults, but not now. First 5.2, then code style discussions ;)

File: pyproject.toml (hunk 2 above)
Comment (gforcada) [reply pyproject.toml]: which two lines? :thinking:  :sweat_smile:

Comment (jensens) [reply pyproject.toml]: lines 23 and 24 in this file - `skip-string-normalization = true` means the same as `-S` on command line.

Comment (gforcada) [reply pyproject.toml]: :sweat_smile: sorry as you pointed to the black docs I thought they were there.

As for these two lines, I would remove the line length one, it is specially annoying for long imports, and as black takes care of it anyway, let's not bang our heads against it (IMHO)

**Pull request closed without merging**